apps/emqx/src/emqx_rpc.erl

```diff
@@ -27,6 +27,8 @@
     cast/5,
     multicall/4,
     multicall/5,
+    multicall_on_running/5,
+    on_running/3,
```

Comment:
Just to confirm: this isn't used anywhere, right?

Reply:
Yes, I guess it can be used only in the next release. But it's good to have it now.